### ISSUE TITLE
Provide a machine-readable way for colspan

### ISSUE THREAD

**Issue by ripla** [vaadin/vaadin-form-layout#101]
Related to #100. `colspan` can be applied to all children, not just `vaadin-form-item`. Document this in a machine-readable way, so that tooling (e.g. Vaadin Designer) can use the property.

**Comment by web-padawan** [comment]
What exactly do you mean by a "machine-readable way"? Are there any existing examples of similar properties for the other components (for setting attribute in arbitrary children)?

Fix for #100 would be trivial but I'm not sure how Designer uses `analysis.json` and therefore it's unclear to me what kind of JSDoc annotation should we use to let it know about this property.

**Comment by ripla** [comment]
Machine readable for me is just some pre-agreed format that is used consistently.

We cannot use `analysis.json` since to my knowledge it's not included in the package.

**Comment by ripla** [comment]
We could also work with a something that we can call at runtime. Currently, we discover the properties of an element `x-foo` by calling `xFoo.constructor.observedAttributes`. We could call an additional function / access a field at runtime to read additional info from the parent.

**Comment by tomivirkki** [comment]
Suggestion: `formLayout.__childAttributes // [{name: 'colspan', type: 'number'},...]`

**Comment by web-padawan** [comment]
The `observedAttributes` only exists for custom elements, while the children of `vaadin-form-layout` can include native HTML elements, like `label` and `input`. Should we handle that somehow?

**Comment by ripla** [comment]
@web-padawan no, that's on us. It's enough if the additional attributes are available in some easily accessible form like the one @tomivirkki suggested above.
